tests/template_tests/test_logging.py

```diff
@@ -69,3 +69,34 @@ def test_log_on_variable_does_not_exist_not_silent(self):
     def test_no_log_when_variable_exists(self):
         Variable('article.section').resolve({'article': {'section': 'News'}})
         self.assertIsNone(self.test_handler.log_record)
+
+
+class ExceptionDuringRenderLoggingTests(SimpleTestCase):
+    def setUp(self):
+        self.test_handler = TestHandler()
+        self.logger = logging.getLogger('django.template')
+        self.original_level = self.logger.level
+        self.logger.addHandler(self.test_handler)
+        self.logger.setLevel(logging.WARN)
+
+    def tearDown(self):
+        self.logger.removeHandler(self.test_handler)
+        self.logger.level = self.original_level
```

Comment:
There's a lot of duplication between this test case and `VariableResolveLoggingTests.setUp` and `tearDown`.

I think it would make sense to create a common base for both of them.

e.g.
```python
class TemplateLoggingTestCase(SimpleTestCase):
    def setUp(self):
        self.test_handler = TestHandler()
        self.logger = logging.getLogger('django.template')
        self.original_level = self.logger.level
        self.logger.addHandler(self.test_handler)
        self.logger.setLevel(logging.DEBUG)

    def tearDown(self):
        self.logger.removeHandler(self.test_handler)
        self.logger.level = self.original_level
```

And assert `self.test_handler.log_record.level == logging.WARN`.